tests/settings/SectionFillingTest.py

```diff
@@ -61,6 +62,19 @@ def test_fill_settings(self):
         self.assertEqual(bool(self.section['use_spaces']), True)
         self.assertEqual(len(self.section.contents), 3)
 
+    def test_fill_settings_aspects(self):
+        sections = {'test': self.section}
+        sections['test'].append(Setting('aspects', 'commitmessage'))
+        sections['test'].append(Setting('language', 'py'))
+
+        _, _ = fill_settings(sections,
```

Comment:
if you are not using any of the returned values, you dont need to store them in `_`